typings/resin-sdk.d.ts

```diff
@@ -244,6 +245,19 @@ declare namespace ResinSdk {
 		is_depended_on_by__application: ReverseNavigationResource<Application>;
 	}
 
+	interface ApplicationType {
+		id: number;
+		name: string;
+		slug: string;
+		supports_gateway_mode: true;
+		supports_multicontainer: true;
+		supports_web_url: true;
```

Comment:
These should surely be `boolean`, not `true`.

Reply:
That's what you get when you write your typings based on retried JSONs :)